lib/active_shipping/package.rb

```diff
@@ -1,118 +1,62 @@
 module ActiveShipping #:nodoc:
   class Package
-    cattr_accessor :default_options
-    attr_reader :options, :value, :currency
+    cattr_accessor :default_options do {} end
+    attr_reader :weight, :length, :width, :height, :options, :value, :currency
 
-    # Package.new(100, [10, 20, 30], :units => :metric)
-    # Package.new(Mass.new(100, :grams), [10, 20, 30].map {|m| Length.new(m, :centimetres)})
-    # Package.new(100.grams, [10, 20, 30].map(&:centimetres))
-    def initialize(grams_or_ounces, dimensions, options = {})
-      options = @@default_options.update(options) if @@default_options
-      options.symbolize_keys!
-      @options = options
+    alias_attribute :mass, :weight
 
-      @dimensions = [dimensions].flatten.reject(&:nil?)
+    def initialize(weight:, length:, width:, height: nil, options: {})
+      @options = @@default_options.merge(options.symbolize_keys)
```

Comment:
If `@@default_options` can be changed by a client of this library then I'd recommend `@@default_options.merge(options).symbolize_keys` to guarantee all keys are symbols.